package.json

```diff
@@ -8,7 +8,7 @@
     "react-scripts": "1.0.14"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "set PORT=2000 && react-scripts start",
```

Comment:
You don't actually need the `set` here 🙂

Reply:
Many thanks for your reviews Ali. I have some other project running on my computer on port 3000 so I did this to change the port for this project.